toqito/state_metrics/bures_distance.py

```diff
@@ -0,0 +1,68 @@
+"""Bures distance metric."""
+import numpy as np
+
+from toqito.state_metrics import fidelity
+
+def bures_distance(rho_1: np.ndarray, rho_2: np.ndarray, decimals: int = 10) -> float:
+    r"""
+    Compute the Bures distance of two density matrices [WikBures]_.
+
+    Calculate the Bures distance between the two density matrices:
```

Comment:
Running the doc generation seems to yield an unexpected indentation warning. This paragraph should be formatted as:

```
    Calculate the Bures distance between the two density matrices: :code:`rho_1` and
    :code:`rho_2`, defined by:
```